lib/new_relic/agent/method_tracer_helpers.rb

```diff
@@ -106,7 +106,7 @@ def namespace_and_location(object, method_name)
         klass = object.singleton_class? ? klassify_singleton(object) : object
         name = klass.name || '(Anonymous)'
         is_class_method = false
-        method = if klass.instance_methods.include?(method_name)
+        method = if (klass.instance_methods + klass.private_instance_methods).include?(method_name)
```

Comment:
I ran some microbenchmarking on different ways to combine arrays, and `+` was the fastest one, but not by much, all but one said `same-ish: difference falls within error`